Cargo.toml

```diff
@@ -55,7 +54,7 @@ nightly = []
 bench = []
 # Use this to enable SSE4.2 instructions in boundary finding
 # TODO: Benchmark this
-sse4 = ["nightly", "twoway/pcmp"]
+sse4 = ["nightly"]
```

Comment:
We can probably drop this feature altogether if `twoway` doesn't need it anymore.